trackma/ui/gtk/mainview.py

```diff
@@ -223,11 +223,11 @@ def _notebook_handle_events(self, widget, event):
         elif self._hovering_over_tabs and event.type == Gdk.EventType.SCROLL:
             page = self.notebook.get_current_page()
             npage = self.notebook.get_n_pages() - 1
-            if event.delta_y < 0 and page > 0:
+            if event.delta_y < 0 < page:
                 self.notebook.prev_page()
             elif event.delta_y > 0 and page < npage:
                 self.notebook.next_page()
-            return event.delta_y < 0 and page > 0 or event.delta_y > 0 and page < npage
+            return event.delta_y < 0 < page < npage or event.delta_y > 0
```

Comment:
This actually changes the logic of the condition.

Reply:
Are you sure about this? I did this one manually because there were no parentheses.

Reply:
```py
return event.delta_y < 0 and page > 0 or event.delta_y > 0 and page < npage
# simplified
return (A and B) or (C and D)

return event.delta_y < 0 < page < npage or event.delta_y > 0
# exploded
return event.delta_y < 0 and 0 < page and page < npage or event.delta_y > 0
# simplified
return (A and B and D) or C
```

In fact, I'd argue that transiently comparing delta_y with page is misleading for semantics because they are not actually related here and just happen to be compared to the same constant (`0`). Same for line 226.

Reply:
That's fair. I'll revert these and rebase on master.